src/__init__.py

```diff
@@ -341,11 +341,11 @@ def add_filler_items(self, item_pool, traps):
 
         return item_pool
 
-    def get_item_counts(self, player: Optional[int] = None, reset: bool = False) -> dict:
+    def get_item_counts(self, player: Optional[int] = None, reset: bool = False) -> dict[str, int]:
         """returns the player real item count"""
         if player == None:
             player = self.player
-        if player not in self.item_counts or reset:
+        if not self.item_counts.get(player, {}) or reset:
```

Comment:
This change was made so if creator call get_item_countsbefore the items are created it doesnt stop the count from generating later when required

Reply:
Fun